modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -325,63 +325,51 @@ private void _addManyToOneRelatedObjectEntries(
-						if (FeatureFlagManagerUtil.isEnabled("LPS-183882")) {
-							SystemObjectDefinitionManager
-								systemObjectDefinitionManager =
-									_systemObjectDefinitionManagerRegistry.
-										getSystemObjectDefinitionManager(
-											objectDefinition.getName());
-
-							BaseModel<?> baseModel =
-								systemObjectDefinitionManager.
-									getBaseModelByExternalReferenceCode(
-										systemObjectDefinitionManager.
-											getBaseModelExternalReferenceCode(
-												primaryKey),
-										objectDefinition.getCompanyId());
-
-							Map<String, Object> values =
-								ObjectEntryDTOConverterUtil.toValues(
-									baseModel,
-									dtoConverterContext.
-										getDTOConverterRegistry(),
-									objectDefinition.getName(),
-									_systemObjectDefinitionManagerRegistry,
-									dtoConverterContext.getUser());
-
-							if (MapUtil.isNotEmpty(values)) {
-								ObjectField objectField =
-									_objectFieldLocalService.fetchObjectField(
+						SystemObjectDefinitionManager
+							systemObjectDefinitionManager =
+								_systemObjectDefinitionManagerRegistry.
+									getSystemObjectDefinitionManager(
+										objectDefinition.getName());
+
+						BaseModel<?> baseModel =
+							systemObjectDefinitionManager.
+								getBaseModelByExternalReferenceCode(
+									systemObjectDefinitionManager.
+										getBaseModelExternalReferenceCode(
+											primaryKey),
+									objectDefinition.getCompanyId());
+
+						Map<String, Object> values =
+							ObjectEntryDTOConverterUtil.toValues(
+								baseModel,
+								dtoConverterContext.getDTOConverterRegistry(),
+								objectDefinition.getName(),
+								_systemObjectDefinitionManagerRegistry,
+								dtoConverterContext.getUser());
+
+						if (MapUtil.isNotEmpty(values)) {
+							ObjectField objectField =
+								_objectFieldLocalService.fetchObjectField(
+									objectDefinition.getTitleObjectFieldId());
+
+							if (objectField == null) {
+								objectField =
+									_objectFieldLocalService.getObjectField(
 										objectDefinition.
-											getTitleObjectFieldId());
-
-								if (objectField == null) {
-									objectField =
-										_objectFieldLocalService.getObjectField(
-											objectDefinition.
-												getObjectDefinitionId(),
-											"id");
-								}
-
-								values.put(
-									objectField.getName(),
-									ObjectEntryValuesUtil.getTitleFieldValue(
-										objectField.getBusinessType(),
-										baseModel.getModelAttributes(),
-										objectField,
-										dtoConverterContext.getUser(), values));
+											getObjectDefinitionId(),
+										"id");
 							}
 
-							relatedObjectEntryAtomicReference.set(
-								(Serializable)values);
-						}
-						else {
-							relatedObjectEntryAtomicReference.set(
-								(Serializable)
-									_objectEntryLocalService.
-										getSystemModelAttributes(
-											objectDefinition, primaryKey));
```

Comment:
@devpedrotavares you need to keep the old behavior otherwise it will cause a breaking change

Reply:
There are users using the API expecting the `SystemModelAttributes`, so we can't change that.